apis/python/src/tiledbsoma/_tiledb_array.py

```diff
@@ -65,9 +65,14 @@ def schema(self) -> pa.Schema:
         Lifecycle:
             Experimental.
         """
-        return tiledb_schema_to_arrow(self._tiledb_array_schema(), self.uri, self._ctx)
+        if isinstance(self._tiledb_array_schema(), tiledb.ArraySchema):
+            return tiledb_schema_to_arrow(
+                self._tiledb_array_schema(), self.uri, self._ctx
+            )
+        else:
```

Comment:
nit: no else needed after a return

Reply:
Changed it. Is it better for readability or something to not have the else here?

Reply:
yeah, it reduces redundancy and drops an indent level, as well as making it obvious that you cannot get out of the if-block